bitwarden_license/src/Commercial.Core/AdminConsole/Services/ProviderService.cs

```diff
@@ -38,13 +38,14 @@ public class ProviderService : IProviderService
     private readonly IOrganizationService _organizationService;
     private readonly ICurrentContext _currentContext;
     private readonly IStripeAdapter _stripeAdapter;
+    private readonly IFeatureService _featureService;
 
     public ProviderService(IProviderRepository providerRepository, IProviderUserRepository providerUserRepository,
```

Comment:
In the `ProviderService`, I updated the `CreateOrganizationAsync` method to pivot off the `enable-consolidated-billing` feature flag. When it's On, it invokes the `OrganizationService.SignupClientAsync` instead of `SignupAsync`.